### PR TITLE
Diagram for ptref_grammar updated

### PR DESCRIPTION
**Pull request by kadhikari**
Jira: https://jira.kisio.org/browse/NAVITIAII-3622

### CODE REVIEW

**Review by SGrenet — APPROVED** [review]

**Review by pbougue — COMMENTED** [review]
This misses the update of the graph source file, no?

**Comment by kadhikari** [comment]
> This misses the update of the graph source file, no?

Thanks @pbougue for information. I will update it.

File: documentation/rfc/ptref_grammar.md [hunk 1]
```diff
@@ -174,7 +174,8 @@ The graph is as follow (edges may have weight to avoid unwanted behavior):
...
+) (use _circo_ engine and format as _png_)
```
Comment (pbougue) [inline documentation/rfc/ptref_grammar.md:178]:
```suggestion
) (use _circo_ engine and format as _png_, then also update the link here)
```

**Review by pbougue — APPROVED** [review]
I only checked the "documentation" is updated everywhere it should.
I let the team deal with the correctness of the graph :wink:

**Pull request merged**